Dockerfiles/agent/Dockerfile

```diff
@@ -72,9 +73,8 @@ EXPOSE 8125/udp 8126/tcp
 # Extra conf.d and checks.d
 VOLUME ["/conf.d", "/checks.d"]
 
-# Placeholder healthcheck, to be improved
-HEALTHCHECK --interval=5m --timeout=20s --retries=1 \
-  CMD ["/opt/datadog-agent/bin/agent/agent", "status"]
+HEALTHCHECK --interval=2m --timeout=5s --retries=2 \
+  CMD ["/probe.sh"]
```

Comment:
agent5 already ships `/probe.sh`. Keeping this would allow us:
  - to ease the migration path (no need to change the liveliness probes in all orchestrator packages)
  - to make it easier to change that in the future